org.freedesktop.Sdk.Extension.deno.yaml

```diff
@@ -0,0 +1,61 @@
+id: org.freedesktop.Sdk.Extension.deno
+branch: '23.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+ prefix: /usr/lib/sdk/deno
+ env:
+    PATH: /usr/lib/sdk/deno/bin:/app/bin:/usr/bin
+modules:
+  # Required for deno to build
+  - name: protobuf
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=None
+      - -DBUILD_SHARED_LIBS=ON
```

Comment:
Why enable shared?

Reply:
no idea, I copied protobuf compilation from somewhere else, I'll try without it